packages/forklift-console-plugin/src/modules/Providers/utils/components/CertificateUpload/FetchCertificateModal.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, useState } from 'react';
+import { calculateThumbprint, useTlsCertificate } from 'src/modules/Providers/hooks';
+import { useModal } from 'src/modules/Providers/modals';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { Loading } from '@kubev2v/common';
+import { Alert, Button, Modal, ModalVariant } from '@patternfly/react-core';
+
+import { VerifyCertificate } from './VerifyCertificate';
+
+export const FetchCertificateModal: FC<{
+  url: string;
+  existingCert: string;
+  handleSave: (cert: string) => void;
+}> = ({ existingCert, url, handleSave }) => {
+  const { toggleModal } = useModal();
+  const { t } = useForkliftTranslation();
```

Comment:
Note (not for this PR):
lib component can  not use `t` and `useForkliftTranslation`, instead the strings can be part of the props.
this component is now a part of the app, so we can fix it if we move to the components lib